_common/qiskit/benchmark_runner.py

```diff
@@ -93,21 +93,20 @@
 
     args = parser.parse_args()
 
-# For Inserting the Noise model default into exec option as it is function call 
+    # For Inserting the Noise model default into exec option as it is function call 
     if args.noise_model != None :
         
         if args.noise_model == 'None':
            args.exec_options["noise_model"] = None 
         else:
             module,method = args.noise_model.split(".")
-            module = globals()[module]
+            module = importlib.import_module(f"custom.{module}")
             method = method.split("(")[0]
```

Comment:
I'm not really crazy about this split approach. It seems like it would be particularly fragile. I wonder if we should just require the user provide these two items separately so we're not doing any fragile string acrobatics?